src/components/Calculator.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import CalcKey from './calcComponents/CalcKey';
+import Display from './calcComponents/Display';
+
+import './Calculator.css';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { result: 0 };
+    this.buttons = [
+      {
+        text: 'AC',
+        type: 'other',
+      },
+      {
+        text: '+/-',
+        type: 'other',
+      },
+      {
+        text: '%',
+        type: 'other',
+      },
+      {
+        text: '÷',
+        type: 'operator',
+      },
+      {
+        text: '7',
+        type: 'other',
+      },
+      {
+        text: '8',
+        type: 'other',
+      },
+      {
+        text: '9',
+        type: 'other',
+      },
+      {
+        text: 'x',
+        type: 'operator',
+      },
+      {
+        text: '4',
+        type: 'other',
+      },
+      {
+        text: '5',
+        type: 'other',
+      },
+      {
+        text: '6',
+        type: 'other',
+      },
+      {
+        text: '-',
+        type: 'operator',
+      },
+      {
+        text: '1',
+        type: 'other',
+      },
+      {
+        text: '2',
+        type: 'other',
+      },
+      {
+        text: '3',
+        type: 'other',
+      },
+      {
+        text: '+',
+        type: 'operator',
+      },
+      {
+        text: '0',
+        type: 'other span-2-col',
+      },
+      {
+        text: '.',
+        type: 'other',
+      },
+      {
+        text: '=',
+        type: 'operator',
+      },
+    ];
+  }
+
+  render() {
+    const { result } = this.state;
+    return (
+      <div className="calc-container">
```

Comment:
- [ ] In order for your design to match the wireframe, kindly adjust the width of the container.  Right now is 100%.
- [ ] OPTIONAL: You could correct the colors with a color picker so they match.

## Explanation

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/158275358-8b2f0c10-e322-4cb0-8739-b5c254ca96b7.png) | ![image](https://user-images.githubusercontent.com/57421823/158275326-063f3f18-4952-4bd8-8fcf-ec01a5604448.png)

## Hints

I manage to correct the width like this:

![image](https://user-images.githubusercontent.com/57421823/158275537-040b8b98-4175-461b-9fa7-2d9a9b4d20cf.png)

And the result:

![image](https://user-images.githubusercontent.com/57421823/158275564-02ad083d-f2c3-417d-a29e-da346875f44a.png)

You could install a "color picker" extension for Chrome or Firefox.  Just activate it, and select the color that you want to know, and it will be copied to the clipboard so you can implement it on your project! 💪

Reply:
Changes done....